vaas-app/src/vaas/router/forms.py

```diff
@@ -43,6 +57,16 @@ class Meta:
             'director': SearchableSelect(),
         }
 
+    def save(self, *args, **kwargs):
+        instance = super().save(*args, **kwargs)
+        instance.save()
+        instance.positive_urls.exclude(url__in=self.cleaned_data['positive_urls']).delete()
+        existing_urls = [p.url for p in instance.positive_urls.all()]
```

Comment:
This for can be replaced by instance.positive_urls.values_list('url', flat=True)
https://docs.djangoproject.com/en/3.0/ref/models/querysets/#django.db.models.query.QuerySet.values_list